packages/ast/src/utils.js

```diff
@@ -1,8 +1,8 @@
 // @flow
 import { signatures } from "./signatures";
 import { traverse } from "./traverse";
-import constants from "@webassemblyjs/helper-wasm-bytecode";
 
+const constants = require("@webassemblyjs/helper-wasm-bytecode");
```

Comment:
I'm confused. We're trying to make it possible for folks to consume these utilities as modules, right? This change seems to go against that. Why is this needed?

Reply:
I'm taking advantage of the CJS interop because I don't have an ESM default export and rollup wasn't able to resolve it.

Reply:
Ok I understand now that was wrong, i'm working on the browser compat of `wast-printer`.